var/spack/repos/builtin/packages/bazel/package.py

```diff
@@ -97,7 +107,7 @@ class Bazel(Package):
     patch("compile-0.29.patch")
 
     # Disable dependency search
```

Comment:
Do we need a new patch for 7.1+?

Reply:
I don't think so. The CC and CXX variables are defined in the installation environment.

Reply:
Yes, but bazel purges the build environment...